app/models/mvi.rb

```diff
@@ -139,7 +139,7 @@ def add_ids(response)
 
   def response_from_redis_or_service
     do_cached_with(key: user.uuid) do
-      mvi_service.find_profile(user)
+      mvi_service.find_profile(user.identity)
```

Comment:
find_profile takes a user_identity, but it happens to work because of all of the delegations that happen in user. 
<img width="720" alt="Screen Shot 2020-05-13 at 4 59 20 PM" src="https://user-images.githubusercontent.com/19188/81864863-33081c00-953b-11ea-8e6d-1eb7bcf6105d.png">

Reply:
by using user_identity to look up things in mvi we'll make it possible to fallback to mvi values in user